### ISSUE TITLE
Liquid Display Problems (1.10.2 + Prior versions.)

### ISSUE THREAD

**Issue by CoolioWolfus** [lumien231/Random-Things#116]
The liquid always flows sideways and goes from 16 px static sides to 4x8 px flowing sides.

I managed to manually fix this for myself with the 1.8 version however the 1.10.2 version seems to have moved the flow code into it's classes.

As well as reporting the flow modes display issue, once fixed could I request a mode addition:

Static - As is.

1 Right click on a side - Flow away from that side
   Side clicked and opposite static, top bottom and other sides flowing away from clicked edge.
   Top - Static top and bottom, sides flowing down.
   Bottom - Static top and bottom, sides flowing up.

2 Second right click on a side.
   Side clicked and opposite flowing down, other sides flowing away from clicked edge.
   Front - Front and back flowing down, top bottom left right all flowing to back.
   Back - Front and back flowing down, top bottom left right all flowing towards the front..

3 Third right click on a side, or second top or bottom - Return to static mode.

^ I feel this would be a helpful change and enable players to hide a lava/water door and similar with more control of flow direction than is currently present in the Liquid Display block.

Edit:
The block in question looks solid when active, rather than taking the transparency level from the liquid in question.

Edit 2:
Check this block code I wrote for mc 1.8 for the corrected fluid flow...
`{
"ambientocclusion": false,
"textures": {
     "end": "blocks/lava_still",
     "side": "blocks/lava_flow",
"particle": "blocks/lava_still"
},
"elements":[
        {
        "type":"cube",
        "from":[0,0,0],
        "to":[16,16,16],
        "faces": {
            "up":{"uv":[0,0,16,16],"texture":"#end","cullface":"up"},
            "down":{"uv":[0,0,16,16],"texture":"#end","cullface":"down"},
            "north":{"uv":[8,0,0,8],"texture":"#side","cullface":"north"},
            "east":{"uv":[8,0,0,8],"texture":"#side","cullface":"east"},
            "south":{"uv":[8,0,0,8],"texture":"#side","cullface":"south"},
            "west":{"uv":[8,0,0,8],"texture":"#side","cullface":"west"}
        }
    }
]
}`
... 0/16's for static and 0/8's for flowing, or you get a compressed double flow (compare this with liquid display - lava) which doesn't match the vanilla liquids.

Speaking of water it's identical except for:
`"textures": {
        "end": "blocks/water_still",
        "side": "blocks/water_flow",
   "particle": "blocks/water_still"
},`

Basically the lava block is a glowstone block with lava textures and the water block is a glass block with water textures.

Screenshots show the above moveable Lava blocks with real Lava in the middle.
![2015-10-05_23 02 42](https://cloud.githubusercontent.com/assets/21157043/17888518/3ecb9b9a-6923-11e6-9272-b3a864aa6e3f.png)
![2015-10-06_00 01 14](https://cloud.githubusercontent.com/assets/21157043/17888517/3eca6c20-6923-11e6-8d41-969964f08d1e.png)
![2015-10-06_00 01 54](https://cloud.githubusercontent.com/assets/21157043/17888516/3eca5226-6923-11e6-8652-054a81886740.png)

I hope this helps correcting the block flow issue,

Feel free to contact myself for some suggestions for other useful random things - mostly blocks. :D
